README.md

```diff
@@ -2,7 +2,7 @@
 
 <img src="https://raw.githubusercontent.com/rotki/rotki/develop/frontend/app/public/assets/images/rotkehlchen_no_text.png" width="250">
 
-rotki is an open source portfolio tracking, analytics, accounting and tax reporting tool that protects your privacy.  The mission of rotki is to bring transparency into the crypto and financial sectors through the use of open source. Most importantly unlike virtually every other competing service which consists of closed source SaaS onto which you are forced to hand over all your financial data, with rotki your data is stored encrypted locally in your computer. It enables you to take ownership of your financial data!
+rotki is an open source portfolio tracking, analytics, accounting and tax reporting tool that protects your privacy. The mission of rotki is to bring transparency into the crypto and financial sectors through the use of open source. Most importantly, rotki stores your data locally on your computer in an encrypted form, unlike virtually every other competing services which consists of closed source SaaS onto which you are forced to transmit all your financial data. This allows you to take ownership of your financial data!
```

Comment:
Hey @omahs thanks for the PR but I am not sure I prefer the new wording here.

Also the wording is wrong ins competing service**s**, plural is not needed.